docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -12,10 +12,23 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_snapshot_config: configuration/index.adoc
 :url_backups: management/snapshots.adoc
 
+NOTE: To upgrade Lisk Service to a desired version, please check out the specific version with *`git checkout <tag>`*, and then create the Docker image, followed by starting Lisk Service as shown in the commands below.
```

Comment:
How to update Lisk Service should not be a note, it should at least be a separate section, and every step necessary to update Lisk Service should be explained it that very section, and supported, if possible, by code snippets the user can simply copy.